docker-compose.yml

```diff
@@ -30,42 +30,21 @@ services:
       - RUNNER_COMMAND_TO_GET_HOST_IP
       - STORAGE_API_TOKEN_READ_ONLY
       - STORAGE_API_TOKEN_FEATURE_NATIVE_TYPES
+      - RUN_BIGQUERY_TESTS
+      - RUN_EXASOL_TESTS
       - RUN_SYNAPSE_TESTS
+      - RUN_TERADATA_TESTS
```

Comment:
To tu chybělo, takž se ty testy reálně nikdy nespustili.

Exasol je vypnutý přes variables, čekáme na testovací projekt a pak to zapnu https://keboola.atlassian.net/browse/PST-628.
Testy pro ostatní backendy se už pouští.